xdp-dump/tests/test-xdpdump.sh

```diff
@@ -340,6 +349,8 @@ test_multi_pkt()
 
 test_perf_wakeup()
 {
+    assert_kernel_version_greater_or_equal 5.7.0
+
```

Comment:
Same as above here, lines 360-361. Maybe you can write a shell function something like:

PASS_REGEX=conditional_replace(<kernel_version>, PASS_REGEX, "xdp_dispatcher", "xdp_test_prog_long")